config/settings/common.py

```diff
@@ -290,3 +290,5 @@
 CONNECTED_APP_CLIENT_ID = None
 CONNECTED_APP_CLIENT_SECRET = None
 CONNECTED_APP_CALLBACK_URL = None
+
+GITHUB_STATUS_UPDATES_ENABLED = env.bool('GITHUB_STATUS_UPDATES_ENABLED', True)
```

Comment:
I was thinking about this. I think this way makes sense, because in TEST we also want to go down the code path, and instead mock out the task and subsequent api call.